src/components/PostEditor/PostEditor.js

```diff
@@ -566,9 +560,14 @@ export default class PostEditor extends React.Component {
       showFiles,
       showImages,
       addAttachment,
-      postTypes
+      postTypes,
+      fetchLinkPreviewPending
     } = this.props
-
+    // Note: Providing `groupIds` to HyloTipTapEditor would cause
```

Comment:
seems worth filing a ticket about I think?

Reply:
Agreed. I'll add it to my post-game
list.  It's been this way sincerely the beginning of time (only 1 group scoped).